package.json

```diff
@@ -1,213 +1,212 @@
-    "@jitsi/eslint-config": "4.1.10",
-    "@types/amplitude-js": "8.16.5",
-    "@types/audioworklet": "0.0.29",
-    "@types/dom-screen-wake-lock": "1.0.1",
-    "@types/js-md5": "0.4.3",
-    "@types/lodash": "4.14.182",
-    "@types/moment-duration-format": "2.2.6",
-    "@types/offscreencanvas": "2019.7.2",
-    "@types/pixelmatch": "5.2.5",
-    "@types/punycode": "2.1.0",
-    "@types/react": "17.0.14",
-    "@types/react-dom": "17.0.14",
-    "@types/react-linkify": "1.0.1",
-    "@types/react-native": "0.69.22",
-    "@types/react-native-keep-awake": "2.0.3",
-    "@types/react-native-video": "5.0.14",
-    "@types/react-redux": "7.1.24",
-    "@types/react-window": "1.8.5",
-    "@types/unorm": "1.3.28",
-    "@types/uuid": "8.3.4",
-    "@types/w3c-image-capture": "1.0.6",
-    "@types/w3c-web-hid": "1.0.3",
-    "@types/zxcvbn": "4.4.1",
-    "@typescript-eslint/eslint-plugin": "5.59.5",
-    "@typescript-eslint/parser": "5.59.5",
-    "babel-loader": "8.2.3",
-    "babel-plugin-optional-require": "0.3.1",
-    "circular-dependency-plugin": "5.2.0",
-    "clean-css-cli": "4.3.0",
-    "css-loader": "6.8.1",
-    "eslint": "8.40.0",
-    "eslint-plugin-import": "2.27.5",
-    "eslint-plugin-jsdoc": "46.2.6",
-    "eslint-plugin-react": "7.32.2",
-    "eslint-plugin-react-native": "4.0.0",
-    "eslint-plugin-typescript-sort-keys": "2.3.0",
-    "jetifier": "1.6.4",
-    "metro-react-native-babel-preset": "0.75.1",
-    "patch-package": "6.4.7",
-    "process": "0.11.10",
-    "sass": "1.26.8",
-    "style-loader": "3.3.1",
-    "traverse": "0.6.6",
-    "ts-loader": "9.4.2",
-    "typescript": "5.0.4",
-    "unorm": "1.6.0",
-    "webpack": "5.76.0",
-    "webpack-bundle-analyzer": "4.4.2",
-    "webpack-cli": "4.9.0",
-    "webpack-dev-server": "4.7.3"
-  },
-  "overrides": {
-    "@xmldom/xmldom": "0.8.7"
-  },
-  "engines": {
-    "node": ">=14.0.0",
-    "npm": ">=7.0.0"
-  },
-  "license": "Apache-2.0",
-  "scripts": {
-    "lint": "eslint --ext .js,.ts,.tsx --max-warnings 0 . && tsc --noEmit",
-    "tsc:web": "tsc --noEmit --project tsconfig.web.json",
-    "tsc:native": "tsc --noEmit --project tsconfig.native.json",
-    "tsc:ci": "npm run tsc:web && npm run tsc:native",
-    "lint:ci": "eslint --ext .js,.ts,.tsx --max-warnings 0 .",
-    "lint:lang": "for file in lang/*.json; do npx --yes jsonlint -q $file || exit 1; done",
-    "lang-sort": "./resources/lang-sort.sh",
-    "lint-fix": "eslint --ext .js,.ts,.tsx --max-warnings 0 --fix .",
-    "postinstall": "patch-package --error-on-fail && jetify",
-    "validate": "npm ls",
-    "tsc-test:web": "tsc --project tsconfig.web.json --listFilesOnly | grep -v node_modules | grep native",
-    "tsc-test:native": "tsc --project tsconfig.native.json --listFilesOnly | grep -v node_modules | grep web",
-    "start": "make dev"
-  },
-  "resolutions": {
-    "@types/react": "17.0.14",
-    "@types/react-dom": "17.0.14"
-  }
+    "name": "jitsi-meet",
```

Comment:
Don't reformat the file please. Keep your changes to a minimum.

Reply:
It doesn't matter where it comes from, we can't accept PRs that reformat files just because.